roles/0-create-local-files/templates/install-config.yaml

```diff
@@ -32,5 +32,11 @@ additionalTrustBundle: |
 imageContentSources:
   - source: quay.io/openshift
     mirrors:
-      - {{ disconnected_registry_url }}
+      - {{ disconnected_registry_url }}/openshift
```

Comment:
Wouldn't it be better to have different variables for different registries? Like disconnected_registry_quai_io, disconnected_registry_registry_access_redhat_com and disconnected_registry_registry.redhat.io.